### ISSUE TITLE
make html fails: Cannot unbox a type object

### ISSUE THREAD

**Issue by pdl** [perl6/doc#241]
After #240, another issue has cropped up later in the build process

```
Composing doc registry ...
Cannot unbox a type object
  in method lookup at /home/d/f/p6-doc/lib/Perl6/Documentable/Registry.pm:26
  in sub MAIN at htmlify.p6:133
  in block <unit> at htmlify.p6:820
```

```
$ perl6 --version
This is rakudo version 2015.11-513-g5308479 built on MoarVM version 2015.11-44-ge7edb06 implementing Perl v6.b.
```

**Comment by sylvarant** [comment]
I think 6eeef84 and bd1480c have fixed this issue?
I no longer encounter this problem

**Comment by jonathanstowe** [comment]
It is definitely generating the docs okay now.

**Comment by moritz** [comment]
For the record, you can check http://doc.perl6.org/build-log/?C=M;O=D to see the logs of the builds that are used to display https://doc.perl6.org/
